lib/archethic/mining.ex

```diff
@@ -80,9 +80,12 @@ defmodule Archethic.Mining do
   @doc """
   Elect validation nodes for a transaction
   """
-  def get_validation_nodes(tx = %Transaction{address: tx_address, validation_stamp: nil}) do
+  def get_validation_nodes(
+        tx = %Transaction{address: tx_address, validation_stamp: nil},
+        ref_timestamp
+      ) do
     current_date = DateTime.utc_now()
-    sorting_seed = Election.validation_nodes_election_seed_sorting(tx, current_date)
+    sorting_seed = Election.validation_nodes_election_seed_sorting(tx, ref_timestamp)
 
     node_list = P2P.authorized_and_available_nodes(current_date)
```

Comment:
Suggestion: you may remove `current_date` since it's the default parameters for `authorized_and_available_nodes`